### PR TITLE
Add external lib for swagger (PR1)

### PR DESCRIPTION
**Pull request by JAmmermann-DLR**
First PR for adding swagger to the REST Server.

Added:
- External lib swagger project, only depenency are the Swagger Annotations
- Some example annotations to the Resources and the Model Classes (will be changed in PR2 anyways)
- License, Notice and about file

Part of #808

### CODE REVIEW

File: de.dlr.sc.virsat.external.lib.swagger/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: VirSat External Lib - Swagger
+Bundle-SymbolicName: de.dlr.sc.virsat.external.lib.swagger
+Bundle-Version: 4.12.1.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-Vendor: DLR (German Aerospace Center)
+Require-Bundle: de.dlr.sc.virsat.external.lib,
+ javax.servlet;visibility:=reexport,
+ de.dlr.sc.virsat.external.lib.jersey;bundle-version="4.12.1"
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.external.lib.swagger/META-INF/MANIFEST.MF:10]: Do we need this dependency? If we do, let's remove the version.

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java [hunk 2]
```diff
@@ -40,14 +42,46 @@
...
+import io.swagger.annotations.SwaggerDefinition;
 
 /**
  * The resource to access the VirSat data model of a server repository
  * Provides an endpoint to access a repository
  */
+@Api(tags = {"Model"})
+@SwaggerDefinition(
+	info = @Info(
+//		description = "Gets the weather",
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java:59]: ?

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java [hunk 3]
```diff
@@ -130,6 +164,21 @@ public Response putProperty(ABeanProperty<?, ?> bean) {
 		@GET
 		@Path(CA + "/{caUuid}")
 		@Produces(MediaType.APPLICATION_JSON)
+
+		@ApiOperation(
+				produces = "application/json",
+				value = "Fetch ca",
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java:170]:
```suggestion
				value = "Fetch CA",
```

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java [hunk 4]
```diff
@@ -130,6 +164,21 @@ public Response putProperty(ABeanProperty<?, ?> bean) {
 		@GET
 		@Path(CA + "/{caUuid}")
 		@Produces(MediaType.APPLICATION_JSON)
+
+		@ApiOperation(
+				produces = "application/json",
+				value = "Fetch ca",
+				httpMethod = "GET",
+				notes = "<br>This service fetches cas",
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java:172]:
```suggestion
				notes = "<br>This service fetches CAs",
```

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java [hunk 5]
```diff
@@ -141,6 +190,14 @@ public Response getCa(@PathParam("caUuid") String caUuid) {
 			}
 		}
 		
+		@ApiOperation(
+				produces = "application/json",
+				value = "Put ca",
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java:195]:
```suggestion
				value = "Put CA",
```

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java [hunk 6]
```diff
@@ -141,6 +190,14 @@ public Response getCa(@PathParam("caUuid") String caUuid) {
 			}
 		}
 		
+		@ApiOperation(
+				produces = "application/json",
+				value = "Put ca",
+				httpMethod = "PUT",
+				notes = "<br>This service puts cas")
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java:197]:
```suggestion
				notes = "<br>This service puts CAs")
```

**Review by andriikovalov-dlr — CHANGES_REQUESTED** [review]
Some minor questions and suggestions

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java (hunk 2 above)
Comment (JAmmermann-DLR) [reply de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java:59]: Forgot to remove the comments, will do it now

File: de.dlr.sc.virsat.external.lib.swagger/META-INF/MANIFEST.MF (hunk 1 above)
Comment (JAmmermann-DLR) [reply de.dlr.sc.virsat.external.lib.swagger/META-INF/MANIFEST.MF:10]: No we don't, thanks :) This was needed when we had more swagger dependencies, but now we don't need it anymore!

**Review by andriikovalov-dlr — APPROVED** [review]

File: de.dlr.sc.virsat.external.lib.swagger/pom.xml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+	<packaging>eclipse-plugin</packaging>
+
+	<!-- Execute with profile downloadLib which is defined in the parent pom 
+		to update all libs and licenses from maven -->
+
+	<properties>
+		<swagger.version>1.6.2</swagger.version>
+	</properties>
+
+	<dependencies>
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.external.lib.swagger/pom.xml:29]: We now wrap these into the downloadLibs profile (check other external libs) to prevents maven from downloading these dependencies during build time.

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Theres a small change in how we handle dependencies in external libs. Needs to be applied here to (super simple just put the dependency tag in the pom in a downloadLibs profile, see other external libs for examples).

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Pull request merged**